api/nodecontrol/auction-response.json

```diff
@@ -4,7 +4,7 @@
   "title": "AuctionResponse",
   "type": "object",
   "properties": {
-    "node_id": {
+    "bidder_id": {
       "type": "string",
       "description": "The unique identifier of the node"
```

Comment:
nit.. this could be something like `A one-time identifier used to target deployments`

Reply:
🤣 Translation: _When you change the field, update the description too dummy_